cluster/common.sh

```diff
@@ -701,6 +701,7 @@ MANIFEST_URL_HEADER: $(yaml-quote ${MANIFEST_URL_HEADER:-})
 NUM_NODES: $(yaml-quote ${NUM_NODES})
 STORAGE_BACKEND: $(yaml-quote ${STORAGE_BACKEND:-etcd2})
 ENABLE_GARBAGE_COLLECTOR: $(yaml-quote ${ENABLE_GARBAGE_COLLECTOR:-})
+MASTER_RESERVED_IP: $(yaml-quote ${MASTER_RESERVED_IP:-})
```

Comment:
As discussed offline I think we should not use env var `MASTER_RESERVED_IP` as it's GCE specific. We should use something that will make sense also on other cloud providers.